src/osemgrep/cli_ci/Ci_CLI.ml

```diff
@@ -115,8 +122,8 @@ let cmdline_term : conf Term.t =
     const combine
     $ Scan_CLI.cmdline_term ~allow_empty_config:true
     $ o_audit_on $ o_beta_testing_secrets $ o_code $ o_dry_run $ o_secrets
-    $ o_supply_chain $ o_suppress_errors $ Git_metadata.env
-    $ Github_metadata.env)
+    $ o_internal_ci_scan_results $ o_supply_chain $ o_suppress_errors
```

Comment:
Actually here it's important to have the order of the parameters to combine above to match exactly the order of those o_xxx arguments here. To facilitate this, we actually sort those parameters alphabetically, so here you can put o_internal_xxx after o_dr... and before o_se...